astropy/stats/sigma_clipping.py

```diff
@@ -154,7 +160,7 @@ class SigmaClip:
 
         .. _bottleneck:  https://github.com/pydata/bottleneck
 
-    stdfunc : {'std'} or callable, optional
+    stdfunc : {'std', 'mad_std'} or callable, optional
```

Comment:
Slightly unrelated, but the text below mentions `bottleneck`, but not the built-in fast method.

Reply:
I've tried to clarify this